controller/gen/apis/policy/v1alpha1/httproute.go

```diff
@@ -441,11 +441,13 @@ const (
 //
 // ```
 // match:
-//   path:
-//     value: "/foo"
-//   headers:
-//   - name: "version"
-//     value "v1"
+//
+//	path:
+//	  value: "/foo"
+//	headers:
+//	- name: "version"
+//	  value "v1"
+//
```

Comment:
Hmm, I ran into this in a previous PR too -- `gofmt` wants to rewrite this comment code block, but it's not a valid / helpful rewrite. And it no longer matches the same comment code block in the v1beta3 file, here:

https://github.com/linkerd/linkerd2/blob/213f123a02e59005c067835f281cc3766804492e/controller/gen/apis/policy/v1beta3/types.go#L486-L494

I recommend reverting all of the `gofmt` changes in this file.